.github/workflows/ci.yml

```diff
@@ -383,6 +383,9 @@ jobs:
           # this suite ignores is not meant to be run with different configs
           - config: default
             suite: suite-oauth2
+          # this suite ignores is not meant to be run with different configs
```

Comment:
that's not English (i know who wrote that...)

maybe remove "ignores"

Reply:
Will extract to preparatory commit.